examples/toolbar/toolbar.html

```diff
@@ -44,32 +46,63 @@ <h1>Toolbar Example</h1>
     <h2 id="ex_label">Example</h2>
     <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
     <div id="ex1">
-        <div class="toolbar" role="toolbar">
-            <div class="toolbar-item selected" role="button" tabindex="0">
-                Tool 1
+        <div class="toolbar" role="toolbar" aria-label="Toolbar">
+          <div class="group characteristics">
+            <div role="button" class="toolbar-item bold" aria-pressed="false" value="bold" tabindex="0">
+              <i class="fas fa-bold"></i>
             </div>
-            <div class="toolbar-item" role="button" tabindex="-1">Tool 2</div>
-            <div class="toolbar-item" role="button" tabindex="-1">Tool 3</div>
-            <div class="toolbar-item" role="button" tabindex="-1">Tool 4</div>
-            <div class="toolbar-item" role="button" tabindex="-1">Tool 5</div>
-            <div class="menu-wrapper">
-              <button
-                tabindex="-1"
-                aria-haspopup="true"
-                aria-controls="menu1"
-                class="toolbar-item menu-button">
-                Menu
-              </button>
-              <ul role="menu" id="menu1">
-                <li role="menuitem">Item 1</li>
-                <li role="menuitem">Item 2</li>
-                <li role="menuitem">Item 3</li>
-                <li role="menuitem">Item 4</li>
-                <li role="menuitem">Item 5</li>
-                <li role="menuitem">Item 6</li>
-              </ul>
+            <div role="button" class="toolbar-item italic" aria-pressed="false" value="italic" tabindex="1">
+              <i class="fas fa-italic"></i>
+            </div>
+            <div role="button" class="toolbar-item underline" aria-pressed="false" value="underline" tabindex="1">
+              <i class="fas fa-underline"></i>
+            </div>
+          </div>
+          <div class="group align">
+            <div role="button" class="toolbar-item textAlign" aria-pressed="true" value="left" tabindex="1">
+              <i class="fas fa-align-left"></i>
```

Comment:
These alignment buttons work like radio so cannot be toggles. Use a radio group or alternatively, use a popup menu with menuitem radios.